app/change_sets/archival_media_collection_change_set.rb

```diff
@@ -15,6 +15,7 @@ class ArchivalMediaCollectionChangeSet < ChangeSet
   property :depositor, multiple: false, required: false, virtual: true
   property :read_groups, multiple: true, required: false
   property :change_set, require: true, default: "archival_media_collection"
+  property :embargo_date, multiple: false, required: false, type: Valkyrie::Types::Date
```

Comment:
It seems like `Valkyrie::Types::Date` isn't doing anything:

```ruby
Valkyrie::Types::Date['2022/01/02'] # => '2022/01/02'
Valkyrie::Types::Date['bla'] # => 'bla'
```

Since that's the case, I suggest we do one of two things:

1. Use `Types::Params::Date` (might be tough, I don't think it accepts 01/02/2022)
2. Use `Valkyrie::Types::String` and add a validation that it's a properly formatted date.